nameko_stripe/pytest.py

```diff
@@ -0,0 +1,70 @@
+import json
+import time
+
+import pytest
+import stripe
+
+from nameko_stripe import constants
+
+
+@pytest.fixture
+def stripe_config(web_config):
+    config = web_config.copy()
+    config.update(
+        {
+            "STRIPE": {
+                "SECRET_KEY": "sk_test_EWX...FbN",
+                "ENDPOINT_SECRET": "whsec_6Yi...Igu",
+            }
+        }
+    )
+    return config
+
+
+@pytest.fixture
+def make_webhook():
+    def make(**fields):
+        payload = {
+            "created": 1326853478,
+            "livemode": False,
+            "id": "evt_00000000000000",
+            "type": "source.chargeable",
+            "object": "event",
+            "request": None,
+            "pending_webhooks": 1,
+            "api_version": "2017-08-15",
+            "data": {},
+        }
+        payload.update(fields)
+        return payload
+
+    return make
+
+
+@pytest.fixture
+def send_webhook(stripe_config, make_webhook, runner_factory, web_session):
+    def send(service_cls, webhook=None, endpoint_secret=None, path=None):
```

Comment:
Please see my other comments about removing `endpoint_secret` from the parameters.

Reply:
Already discussed in other comments.